### ISSUE TITLE
Does <meta> accept extraneous items?

### ISSUE THREAD

**Issue by Yomguithereal** [gephi/gexf#23]
Related to https://github.com/graphology/graphology/issues/497

It seems to me the specs are not particularly precise regarding what the `<meta>` section should accept, but the xsd schema seems to indicate you should only be able to specify `<creator>`, `<description>` and `<keywords>`. What's more, I think I already found gexf in the wild having arbitrary tags there, notably `<title>`.

Should we amend the xsd schema to fit usage? Am I hallucinating this usage? What was meant to be accepted here?

**Comment by duncdrum** [comment]
Don't see a good reason why it shouldn't.

**Comment by mbastian** [comment]
Yes, I think `title` should be added, it's a known gap. I just created an [issue](https://github.com/gephi/gexf/issues/24) to not forget about it. If you see other gaps, by all means suggest them!
